sos/report/plugins/vectordev.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+class Vectordev(Plugin):
+
+    short_desc = 'A lightweight, ultra-fast tool for building observability pipelines'
+
+    plugin_name = "vectordev"
+    profiles = ('observability',)
+
+    conf_file = "/etc/vector/vector.yaml"
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/vector/"
```

Comment:
No credentials or secret keys there, I assume..?

Reply:
Not on our use case, as we don´t  require authentication to our sinks, but the configuration of `vector` allows a bunch of different authentications to destinations, each with their own set of different credential names and settings.
I'm not sure how to deal with this for the time being, but any comment is welcome.

Reply:
Looking at the docs, it seems like these are done via INI configs? If so, I think we can take a reasonable stab at a generic `postproc` for that format, and use some globs for hitting the typical keys like `pass*`, `key`, `token`, etc...

It won't get everything most likely, but it should at least provide _some_ measure of confidence. We'll have to rely on bug reports from there.

Reply:
I captured a couple of keys that seemed more common like: 
"auth.password",
"auth.token",
"tls.key_pass",

and as vector allows both .init and .yaml style configs I adapted the regepx to capture both